server/src/main/java/org/opensearch/index/search/stats/ShardSearchStats.java

```diff
@@ -91,6 +91,9 @@ public void onPreQueryPhase(SearchContext searchContext) {
                 statsHolder.suggestCurrent.inc();
             } else {
                 statsHolder.queryCurrent.inc();
+                if (searchContext.shouldUseConcurrentSearch()) {
+                    statsHolder.concurrentQueryCurrent.inc();
+                }
             }
```

Comment:
@sohami this is good question: query executed concurrently is still a query, I think we should increment the counter. But  with `concurrentQueryCurrent` we could hint which percentage of queries has been executed concurrently. 

Also, I would imagine this query counter could be used in monitoring, so we change the semantic of it, it could basically produce different results (query count drop).